src/metric_model_reporter.cc

```diff
@@ -53,6 +54,10 @@ MetricReporterConfig::ParseConfig(bool response_cache_enabled)
       latency_counters_enabled_ = false;
     }
 
+    if (pair.first == "histogram_latencies" && pair.second == "true") {
```

Comment:
Just an FYI for the future that it would probably make sense to expand histogram support to the existing latency metrics (queue, compute, etc) similar to how the summary metrics work. But it's not in scope for this PR.

For example, the `--metrics-config histogram_latencies=true` should enable this in the future similar to summaries. This would address this ask: https://github.com/triton-inference-server/server/issues/7672

Reply:
Yes. In fact, I am planning on expaniding the following latency metrics to histograms in 24.11.
- nv_inference_request_duration_us
- nv_inference_queue_duration_us